### PR TITLE
{math}[foss/2023a] JAGS v4.3.2

### PR DESCRIPTION
**Pull request by jkuhl-uni**

### CODE REVIEW

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/8c113f79f4225da6a5967896390f6ea9 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/j/JAGS/JAGS-4.3.2-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    ('https://sourceforge.net/projects/mcmc-%(namelower)s/files/%(name)s/%(version_major)s.x/Source/', 'download'),
+]
+sources = [SOURCE_TAR_GZ]
+checksums = ['871f556af403a7c2ce6a0f02f15cf85a572763e093d26658ebac55c4ab472fc8']
+
+configopts = ' --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s', 'libexec/%(namelower)s-terminal', 'lib/libjags.so'],
```
Comment (verdurin) [inline easybuild/easyconfigs/j/JAGS/JAGS-4.3.2-foss-2023a.eb:27]: @jkuhl-uni please revert the removal of `SHLIB_EXT` here - that's the approach we use, which copes with different library naming conventions.

**Review by verdurin — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/j/JAGS/JAGS-4.3.2-foss-2023a.eb (hunk 1 above)
Comment (jkuhl-uni) [reply easybuild/easyconfigs/j/JAGS/JAGS-4.3.2-foss-2023a.eb]: Sure, thanks for the hint. I gained this EC from --try-toolchain and didn't pay attention, sorry.

File: easybuild/easyconfigs/j/JAGS/JAGS-4.3.2-foss-2023a.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    ('https://sourceforge.net/projects/mcmc-%(namelower)s/files/%(name)s/%(version_major)s.x/Source/', 'download'),
+]
+sources = [SOURCE_TAR_GZ]
+checksums = ['871f556af403a7c2ce6a0f02f15cf85a572763e093d26658ebac55c4ab472fc8']
+
+configopts = ' --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK"'
+
+
+sanity_check_paths = {
+    'files': ['bin/%(namelower)s', 'libexec/%(namelower)s-terminal', 'lib/libjags.so' % SHLIB_EXT],
```
Comment (verdurin) [inline easybuild/easyconfigs/j/JAGS/JAGS-4.3.2-foss-2023a.eb:27]:
```suggestion
    'files': ['bin/%(namelower)s', 'libexec/%(namelower)s-terminal', 'lib/libjags.%s' % SHLIB_EXT],
```

**Review by verdurin — CHANGES_REQUESTED** [review]

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
easybuild-c7.novalocal - Linux CentOS Linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/verdurin/587a6830dca7392b5b39b8b44b21d0a4 for a full test report.

**Comment by verdurin** [comment]
@boegelbot please test @ generoso

**Comment by jkuhl-uni** [comment]
What is missing here @verdurin? I thought the only thing missing was the `SHLIB_EXT` replacements.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3117.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/95a3254b728fc62435213de10cf7c64f for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jkuhl-uni!

**Pull request merged**